examples/nlp/language_modeling/run_LDDL_bert.sh

```diff
@@ -0,0 +1,9 @@
+HYDRA_FULL_ERROR=1 torchrun --nnode 1 --nproc_per_node 4 megatron_bert_pretraining.py \
```

Comment:
Can we also add some README , on why use LDDL and when to use it ?

Reply:
Sorry I wasn't planning on actually commiting this script.  I was just using it for testing.  I can add a readme on using LDDL